interp/interpreter.go

```diff
@@ -493,8 +493,8 @@ func (r *runner) run(fn *function, params []value, parentMem *memoryView, indent
 				// possibly trying to recover from it if it failed to execute.
 				if r.debug {
 					argStrings := make([]string, len(operands))
```

Comment:
This should be `len(operands)-1`, I think, or you get a trailing comma, unless that's desired for some reason.

Reply:
Oh, you're right. I fixed the out-of-bounds crash but didn't then check the output was correct.

Reply:
Fixed off-by-one in the off-by-one fix.